default.env

```diff
@@ -3,10 +3,10 @@
 # https://docs.docker.com/compose/environment-variables/#the-env-file
 # environmental variables for interpolation in docker-compose YAML files
 
-PGRST_JWT_SECRET=<generate via `openssl rand -base64 32`>
+PGRST_JWT_SECRET=<generate via 'openssl rand -base64 32'>
```

Comment:
please revert this change - we want the value of executing that command, not the string 'openssl rand -base64 32'

hint: try both on command line and then `echo $PGRST_JWT_SECRET`